source/js/components/join/join.jsx

```diff
@@ -53,7 +57,7 @@ export default class JoinUs extends React.Component {
       navWrapper.querySelector(".form-dismiss") &&
       this.state.apiSuccess
     ) {
-      navWrapper.querySelector(".form-dismiss").textContent = "Back to menu";
+      navWrapper.querySelector(".form-dismiss").textContent = i18n._(t`Back to menu`);
```

Comment:
is there an equivalent function that isn't `_` ? For Django, `_` might be the convention, but in JS land that's the "underscore" library, which we don't use, nor want to create the impression we are.

Reply:
Also, does this _need_ the `t` template tag? It seems odd to need a function as well as an implicit function for the only argument.

Reply:
I don’t think we can change `i18n._`, unless we can override `@lingui/react`

`t` is a macro, c.f. https://lingui.js.org/ref/macro.html & https://github.com/stereobooster/react-lingui-example#translate-text-without-trans-macro

So we could definitely choose not to use the macro, and write `i18n._(/*i18n*/{id: "Back to menu"})` instead. But it’s arguably less elegant…

Reply:
Oh, actually, it looks like ``i18n.t`Back to menu` `` works just fine!

Reply:
Such a roller-coaster. This has been deprecated in v3: https://lingui.js.org/releases/migration-3.html

![image](https://user-images.githubusercontent.com/1294206/70081284-21236400-1600-11ea-9d42-744082856fdc.png)